cmd/env/push/push.go

```diff
@@ -95,19 +105,15 @@ func RunPush(args []string) error {
 	}
 
 	// If not a monorepo, proceed with regular push
-	return pushForMember(args, secret.SecretKeyId)
+	return pushForMember(args, secret, cmd)
 }
 
 // pushForMember contains the original push logic
-func pushForMember(args []string, secretKeyId int64) error {
+func pushForMember(args []string, secret models.Secret, cmd *cobra.Command) error {
 	config, err := config.RestoreConfig()
 	if err != nil {
 		return err
 	}
-	secret, err := secret.LoadSecret(config.OrganizationId, *config.ProjectId, true)
```

Comment:
the old logic here would always load the secret after also getting a secret key ID. I found it confusing and changed this flow a bit.